client/src/Application.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import CssBaseline from '@mui/material/CssBaseline';
+import Container from '@mui/material/Container';
+import Box from '@mui/material/Box';
+import GitHubTokenForm from './auth/GitHubTokenForm';
+import GitUserActivityPage from './GitUserActivityPage';
+import GitHubApolloProvider from './auth/GitHubApolloProvider';
+import GitHubServerNav from './auth/GitHubServerNav';
+
+const Application: React.FC = ({ children }) => {
+  return (
+    <Box sx={{ backgroundColor: 'rgb(240, 240, 240)', minHeight: '100vh' }}>
+      <CssBaseline />
+      <Container>
+        <Routes>
+          <Route path="/" element={<GitHubServerNav />}>
+            <Route path="new" element={<GitHubTokenForm />} />
+            <Route
```

Comment:
Is the idea to allow multiple git repositories at the same time and treat them similar to organizations that you can switch to?

Minor suggestion - while doing this, what do you think or changing main route to `/user`? So the route would be something like:
`/api.github.com/user?author=bbarbosa`

This would allow adding other routes eventually.

Reply:
I thought about changing but wanted to limit the scope of changes for this PR.

I was thinking we could use `/api.github.com/users/bbarbosa` and use the query params for the dates. Later we could add `/api.github.com/teams/some-github-team`

Reply:
I like that. Agree it's an easy enough change later.